app/views/submissions/show.html.haml

```diff
@@ -110,3 +111,4 @@
             = icon_to 'fa-pencil-square-o', edit_review_path(review), title: "Edytuj recenzję"
```

Comment:
Tu by się przydał przycisk akceptacji, który wywoływał by akcję `accept` - żeby można to było przetestować.

Przypominam również o teście.